components/image-header.tsx

```diff
@@ -29,12 +29,12 @@ export default function ImageHeader({
 
       <section
         className={cn(
-          'absolute flex w-full bg-cover bg-top max-sm:static',
+          'absolute flex w-full bg-neutral-700 bg-cover bg-center bg-blend-overlay max-sm:static',
```

Comment:
The bg will be hidden once the image loads in, right?

Reply:
thats jus a small gradient on top of the image for better text visibililty, like how its done in slideshow